.devcontainer/devcontainer.json

```
@@ -9,8 +9,13 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/universal
 {
+    // Use the image from the CI
     "image": "ghcr.io/gpuweb/gpuweb:main",
-
+    // Use this instead if you need to build the image from the local Dockerfile:
+    // "build": {
+    //     "dockerfile": "../tools/custom-action/Dockerfile",
+    //     "context": ".."
+    // },
```

nit: blank line here so it's clear where the commented out alternative ends
```suggestion
    // },

```